src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -1291,6 +1300,8 @@ def notification_push(endpoint_type, conn, account=None):
 
     # create objects in the bucket
     number_of_objects = 100
+    if cloudevents:
```

Comment:
why is this different?

Reply:
It was 10 before, and cloudevents misses some object if we increase the number

Reply:
cloudevents are just some extra http headers. if we are missing events there might be an issue.
lets investigate that